packages/now-cli/test/dev/integration.js

```diff
@@ -291,7 +294,30 @@ function testFixtureStdio(directory, fn) {
       });
 
       await readyResolver;
-      const helperTestPath = (...args) => testPath(t, port, ...args);
+
+      let deploymentUrl;
+      if (testLiveDeployment) {
+        const token = await fetchTokenWithRetry();
+        const event = await deployWithNowClient({
```

Comment:
@TooTallNate I know we talked about using `now-client` to deploy these tests, but I'm getting the error 

```
The `projectSettings` object is required for new projects, but is missing in the deployment payload
```

So it makes me think we should use `now-cli` instead because that will do the auto-detection and then we can remove the `__NOW_SKIP_DEV_COMMAND` workaround (I think @lucleray added that) because we'll be linked to a project when we run `now dev`.

Thoughts?

Reply:
I'm going to attempt to write the project settings from the response to a local `.now` directory and see if that works. Might not need now cli for this.